Core/HLE/sceMp3.cpp

```diff
@@ -397,30 +402,41 @@ static int sceMp3Init(u32 mp3) {
 	// Parse the Mp3 header
 	int layerBits = (header >> 17) & 0x3;
 	int versionBits = (header >> 19) & 0x3;
-	ctx->SamplingRate = __CalculateMp3SampleRates((header >> 10) & 0x3, versionBits);
-	ctx->Channels = __CalculateMp3Channels((header >> 6) & 0x3);
-	ctx->BitRate = __CalculateMp3Bitrates((header >> 12) & 0xF, versionBits, layerBits);
-	ctx->MaxOutputSample = CalculateMp3SamplesPerFrame(versionBits, layerBits);
-	ctx->freq = ctx->SamplingRate;
+	int bitrate = __CalculateMp3Bitrates((header >> 12) & 0xF, versionBits, layerBits);
+	int samplerate = __CalculateMp3SampleRates((header >> 10) & 0x3, versionBits);;
+	int channels = __CalculateMp3Channels((header >> 6) & 0x3);
 
-	DEBUG_LOG(ME, "sceMp3Init(): channels=%i, samplerate=%iHz, bitrate=%ikbps, layerBits=%d ,versionBits=%d,HEADER: %08x", ctx->Channels, ctx->SamplingRate, ctx->BitRate, layerBits, versionBits, header);
+	DEBUG_LOG(ME, "sceMp3Init(): channels=%i, samplerate=%iHz, bitrate=%ikbps, layerBits=%d ,versionBits=%d,HEADER: %08x", channels, samplerate, bitrate, layerBits, versionBits, header);
 
 	if (layerBits != 1) {
 		// TODO: Should return ERROR_AVCODEC_INVALID_DATA.
```

Comment:
@unknownbrackets Will you test this case ?

Reply:
I've tested it, including now with newer firmware.  This is the error code it returns on real firmware.

That said, I was worried that there might be more to it or cases, or that it might be making someone's custom MP3s work.  It didn't seem important to block so I just added reporting.

It gets reported a lot but I think it's either game hacks or custom music files:
https://report.ppsspp.org/logs/kind/1041
(most of the games are PES hacks.)

-[Unknown]